installer/image_push/tasks/main.yml

```diff
@@ -0,0 +1,40 @@
+---
+- name: Remove local images to ensure proper push behavior
+  block:
```

Comment:
You might add a `TODO` here and link to the future `docker_image` PR so we'll have some visibility to fix it up.